frame/contracts/src/lib.rs

```diff
@@ -373,6 +378,10 @@ pub trait Trait: frame_system::Trait + pallet_transaction_payment::Trait {
 
 	/// The maximum size of a storage value in bytes.
 	type MaxValueSize: Get<u32>;
+
+	/// Used to supply contracts with the current gas price. This is **not** used to
```

Comment:
Prob. we should put it as "used to answer contracts' queries regarding the current gas/weight price"

Reply:
That is better. Done.